libc/src/__support/FPUtil/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ add_header_library(
   HDRS
     FEnvImpl.h
   DEPENDS
-    libc.include.fenv
```

Comment:
We still need to have `libc.include.fenv` in the dependency for this target for now.  We can remove it once `FEnvImpl.h` does not need `#include <fenv.h>` anymore.